test/unit/logic/transaction.js

```diff
@@ -198,7 +198,7 @@ describe('transaction', () => {
 			inValidTransaction.senderPublicKey =
 				'c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6fx';
 			try {
-				expect(transactionLogic.sign(keyPair, inValidTransaction));
+				expect(transactionLogic.sign(keyPair, inValidTransaction)).to.be.error;
```

Comment:
if it returns error it's
`expect(transactionLogic.sign(keyPair, inValidTransaction)).to.be.an('error')`
and if it throws then
`transactionLogic.sign.bind(null, keyPair, inValidTransaction).to.throw(Error, 'message')`

Reply:
`to.throw` doesn't work but I take your suggestion to remove the expect from here.